pipeline/03-evaluate.R

```diff
@@ -251,8 +259,14 @@ gen_agg_stats <- function(data, truth, estimate, bldg_sqft,
     ) %>%
     mutate(across(
       -(contains("_max") & contains("yoy")) & where(is.numeric),
-      ~ replace(.x, !is.finite(.x), NA)
-    ))
+      ~ replace(.x, !is.finite(.x), NA),
+      geography_id = ifelse(
+        is.na(geography_id) & geography_type == "triad_code",
+        triad_code,
+        geography_id
+      )
```

Comment:
**[Question, non-blocking]** Isn't this basically what `add_triad_code` is doing? Also, do we need the `geography_type` check in `add_triad_code` too?

Reply:
Whoops, nice catch. I left this in from an earlier iteration of trying to solve this fix. This is why we code review. Fixed in 89de3ae.